src/components/menus/operating-status-menu.js

```diff
@@ -73,7 +73,7 @@ const withBranchMenu =
         const { snackError } = useSnackMessage();
         const isAnyNodeBuilding = useIsAnyNodeBuilding();
         const { getNameOrId } = useNameOrId();
-        const [branch, setBranch] = useState(null);
+        const [equipmentStatus, setEquipmentStatus] = useState(null);
```

Comment:
same here, it's a state for the equipment not a status only, then keep branch name or rename it equipment wich is too large I think.